### PR TITLE
KH-565: Added release notes for version 1.7.0

### PR DESCRIPTION
**Pull request by kdaud**
Ticket: https://mekomsolutions.atlassian.net/browse/KH-565

### CODE REVIEW

**Review by kazlaw — APPROVED** [review]
I am not so sure about the point of removing the curly braces for the CCS form as a feature though. Probably a bug fix as well as setting the gender values to lowercase.

File: README.md [hunk 1]
```diff
@@ -26,9 +26,40 @@ A large part Ozone Kh's roadmap is managed publicly to promote transparency and
...
 ## Release Notes
 
-### Version 1.7.0 (WIP)
+### Version 1.7.0
+
+- Updated project structure to be compatible with Ozone alpha.10 and above.
+- Removed OCL packages and retire Cancer concept.
+- Updated Common Reports to version 1.4.2
+- Fixed Route translation.
+- Included HPV virus test results in Cancer section in CSS form.
```
Comment (suruchee) [inline README.md:35]: @kdaud which ticket are you referring for this note?

File: README.md [hunk 2]
```diff
@@ -26,9 +26,40 @@ A large part Ozone Kh's roadmap is managed publicly to promote transparency and
...
+- Adjusted aggregator artifact ID and remove relativePath.
+- Explicitly specify public repository in modules' pom.xml.
+- Disabled 'base' OpenMRS validation.
+- Validated on PR & self-rebuild upon dependency changes.
+- Compiled a dependency report for 'base' sub-module.
+- Renamed the Nexus server ID.
+- Set address and tags to Locations.
+- Copied OpenMRS configs for validation.
+- Fixed validator config path.
+- Removed empty curly brackets from Cervical Cancer Screening form.
```
Comment (suruchee) [inline README.md:44]: @kdaud, I could not find a ticket for this as well.

File: README.md [hunk 3]
```diff
@@ -26,9 +26,40 @@ A large part Ozone Kh's roadmap is managed publicly to promote transparency and
...
+- Updated translation for 'History of Cervical Cancer Screening' in CCS form.
+- Updated question under Cancer section in CSS form.
```
Comment (suruchee) [inline README.md:46]: @kdaud, which ticket are you referring to for this note?

File: README.md [hunk 4]
```diff
@@ -26,9 +26,40 @@ A large part Ozone Kh's roadmap is managed publicly to promote transparency and
...
+- Updated translation for Abscesses in CCS form.
+- Updated translation for 'Tele-Consultation with NCD unit' in NCD Screening form.
+- Renamed 'Insurance Type' to 'Payment type' in Start visit.
+- Updated translation for 'Have you been screened for Diabetes and Hypertension?' in NCD Screening form.
+- Updated translation for Weight in NCD Screening form.
+- Renamed 'Create New Patient' to 'Register New Patient' in the Patient Registration.
+- Renamed 'HbA1c' to 'HbA1c (optional)' in NCD Screening and NCD Consultation forms.
+- Swapped O3 React Form Engine with AMPATH Form Engine.
+- Created the Superset config folder and Maven sub-module.
+- Fixed gender values to lowercase.
```
Comment (suruchee) [inline README.md:59]: @kdaud where did you take the gender value note from?

File: README.md (hunk 1 above)
Comment (kdaud) [reply README.md]: The note is not part of version 1.7.0. I've updated the notes.

File: README.md (hunk 2 above)
Comment (kdaud) [reply README.md:44]: This is a bug fix when CCS form could not open after upgrading KH Distro to run on O3 beta.20/21. Ticket: https://mekomsolutions.atlassian.net/browse/KH-559

File: README.md (hunk 4 above)
Comment (kdaud) [reply README.md:59]: This fixed gender config values to lowercase. I do not think there is a ticket however, the PR is at https://github.com/mekomsolutions/ozone-distro-cambodia/pull/349

File: README.md (hunk 3 above)
Comment (kdaud) [reply README.md]: KH-509, I've update the note.

**Pull request merged**